charts/grafana-dashboards/istio-teams/service-dashboard.json

```diff
@@ -3,7 +3,10 @@
     "list": [
       {
         "builtIn": 1,
-        "datasource": "-- Grafana --",
```

Comment:
I see all the services BUT I should see only the one belonging to a team
<img width="1833" alt="image" src="https://github.com/redkubes/otomi-core/assets/17126497/84ea19e9-937a-4677-896e-07de6a15b53f">

Reply:
<img width="1870" alt="image" src="https://github.com/redkubes/otomi-core/assets/17126497/f50549a8-346e-490c-940f-3a68bd26d487">

Reply:
istio dashboard has been removed for teams